components/global/core/io.cellery.observability.telemetry.receiver/src/main/java/io/cellery/observability/telemetry/receiver/internal/MetricsHandler.java

```diff
@@ -0,0 +1,71 @@
+package io.cellery.observability.telemetry.receiver.internal;
```

Comment:
No license header. This indicates that your have't configured your IDE to automatically insert the header